doc/readme.md

```diff
@@ -0,0 +1,335 @@
+Eine genauere Anleitung finden Sie [hier](https://github.com/Programmiermethoden/Dungeon/wiki/Import-Project)
+
+Sie können über die run-Funktion Ihrer IDE überprüfen, ob die Installation lauffähig ist. Alternativ können Sie per Konsole in das Dungeon-Verzeichnis wechseln und `./gradlew run` ausführen.
+
+
+*Falls Sie Probleme beim Installieren haben, schauen Sie in die [Kompatibilitätsliste](https://github.com/Programmiermethoden/Dungeon/wiki/JDK-Kompatibilität) und die [FAQ](https://github.com/Programmiermethoden/Dungeon/wiki/FAQ). Melden Sie sich frühzeitig falls Ihr Problem damit nicht behoben werden konnte.*
+
+
+## Grundlagen
+
+Zu Beginn einige grundlegende Prinzipien, die Sie verstanden haben sollten, bevor Sie mit dem Dungeon arbeiten.
+
+Das Dungeon benutzt das Cross-Plattform-Java-Framework [`libGDX`](https://libgdx.com) als Backend.
+Dieses ist im `Dungeon`-Projekt bereits als Abhängigkeit in die Gradle-Konfiguration integriert, Sie müssen dieses nicht extra installieren. Die Ihnen zur Verfügung gestellten Vorgaben sind so umgesetzt, dass Sie kein tieferes Verständnis für `libGDX` benötigen, um die Aufgaben zu lösen. Sollten Sie allerdings einmal auf Probleme stoßen, kann es unter Umständen helfen, einen Blick in die [Dokumentation von `libGDX`](https://libgdx.com/wiki/) zu werfen.
+
+Das `Dungeon`-Projekt fungiert, ganz vereinfacht gesagt, als eine Facade zwischen `libGDX` und Ihrer eigenen Implementierung. Es implementiert ein Entity-Component-System (ECS):
+- Entity: Entitites sind die Elemente (Helden, Monster, Schatzkisten, etc.) des Spiels
+- Component: Components speichern die Datensätze der Entitäten (z.B. die Lebenspunkte)
+- System: Systeme beinhalten die eigentliche Logik und agieren auf die Components
+
+*Weiteres zum ECS im Dungeon erfahren Sie [hier](ecs/ecs_basics.md)*.
+
+Sie selbst nutzen und erweitern die Components und Systeme der Vorgaben. Sie werden ebenfalls neue Entities, Components und Systeme konzeptionieren und implementieren. So erschaffen Sie z.B. Ihre eigenen Monster und fallengespickte Level.
+
+Sie werden im Laufe der Praktika verschiedene Assets benötigen. Diese liegen per Default im `assets`-Verzeichnis. Sie können das Standardverzeichnis in der `build.gradle` anpassen.
+
+- Standardpfad für Texturen: `assets/`
+- Standardpfad für Charaktere: `assets/character/`
+- Standardpfad für Level-Texturen: `assets/textures/dungeon/`
+
+## Strukturen
+
+In diesem Abschnitt werden Ihnen die wichtigsten Klassen im Dungeon vorgestellt.
+
+![Struktur ECS](ecs/img/ecs.png)
+
+Game ist die Basisklasse von der alles ausgeht. Die Methode `Game#render` ist die Game-Loop. Die Klassen `Entity`, `Component` und `ECS_System` sind die Implementierungen des ECS.
+
+Die LevelAPI generiert, zeichnet und speichert das aktuelle Level. Mehr zum Thema Level erfahren Sie [hier](../level/readme.md).
+
+*Anmerkung: Das UML ist für bessere Lesbarkeit gekürzt.*
```

Comment:
Bitte nicht den gesamten Satz in Italics. Es reicht völlig, "Anmerkung" kursiv zu setzen!